exporter/zipkinexporter/zipkin_test.go

```diff
@@ -28,14 +28,14 @@ import (
 	"strings"
 	"testing"
 
-	commonpb "github.com/census-instrumentation/opencensus-proto/gen-go/agent/common/v1"
+	commonpb "github.com/open-telemtery/opentelemetry-proto/gen-go/agent/common/v1"
```

Comment:
I think this won't work, there is no `open-telemtery/opentelemetry-proto`. This should stay with opencensus.
Same everywhere else opentelemetry-proto is referenced.